diff --git a/main.js b/main.js
index 9d31fa5..8856e24 100644
--- a/main.js
+++ b/main.js
@@ -342,8 +342,8 @@ require([
                 }
 
                 // add prefix URL for endpoint
-                if (apiProject.url)
-                    fields.article.url = apiProject.url + fields.article.url;
+                var prefix_url = (fields.article.group == "Callback_URLs__Optional_") ? apiProject.callback_url : apiProject.url;
+                fields.article.url = prefix_url + fields.article.url;
 
                 addArticleSettings(fields, entry);
 
